+sw_tests/+system_tests/systemtest_spinwave_symbolic_nips.m

```diff
@@ -0,0 +1,117 @@
+classdef systemtest_spinwave_symbolic_nips < sw_tests.system_tests.systemtest_spinwave
+
+    properties
+        reference_data_file = '';
+        swobj_nn = [];
+        symspec = [];
+    end
+
+    properties (TestParameter)
+        test_spectra_function_name = {'sw_neutron', 'sw_egrid', 'sw_instrument', ...
+                                      'sw_omegasum', 'sw_plotspec', 'sw_tofres'};
+    end
+
+    methods (TestClassSetup)
+        function prepareForRun(testCase)
+            % Skips all these tests if the symbolic toolbox is not installed
+            testCase.assumeTrue(sw_hassymtoolbox(), 'Symbolic Toolbox not installed');
```

Comment:
Rather than calling `sw_hassymtoolbox` here, will it work if you tag the whole class as symbolic e.g. `classdef (TestTags = {'symbolic'})`?